include/solvers/navier_stokes_base.h

```diff
@@ -320,6 +320,13 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   void
   refine_mesh();
 
+  /**
+   * @brief Allow the initial refinement of all cells of the principal mesh that are partially
+   * contained in one of the cells of the box refinement mesh given in the parameter.
+   */
+  void
+  box_refine_mesh();
+
```

Comment:
Why is box_refine in gls_sharp and box_refine_mesh in the base class?

Reply:
error now corrected